Cargo.toml

```diff
@@ -11,7 +11,7 @@ documentation = "https://docs.rs/system-deps/"
 readme = "README.md"
 
 [dependencies]
-pkg-config = "0.3"
+pkg-config = "0.3.23"
```

Comment:
We could as well use `0.3.27` which is the latest one.

Reply:
There's no point in requiring the latest version unless you actually need it. It only causes potential dependency resolution problems by having a stricter-than-needed dependency.

Reply:
Right, so shouldn't we use `pkg-config = "~0.3.23"` ?

I never actually used https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#tilde-requirements

Reply:
For 0.y.z using tilde is probably ok ... but please don't, it sounds like a guarantee for dependency resolution conflicts in the future